week-1/L-functions-nested/exercise.js

```diff
@@ -0,0 +1,26 @@
+/*function getAgeInDays(age) {
+    return age * 365;
+  }
+  
+  function createGreeting(name, age) {
+    var ageInDays = getAgeInDays(age);
+    var message =
+      "My Name is " + name + " and I was born over " + ageInDays + " days ago!";
+    return message;
+    
+  }
+  var nameAndAge =createGreeting("Saeed",31);
+  console.log(nameAndAge);  */
+
+function percentageOfPpl(a, b) {
+  return Math.round((a / (a + b)) * 100);
+}
+function text(a, b, c) {
+  var perOfPpl = percentageOfPpl(a, b);
+  var message = "Percentage " + c + ":" + perOfPpl + "%";
+  return message;
+}
```

Comment:
Good :+1: . Same comment about the parameter and function names though. If I see a function called 
```
text (a, b, c)
```
somewhere I would have no idea what it does.